### ISSUE TITLE
Solidify our Giveth Turkey donation presence for Devconnect 

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-planning#1342]
The EF is skipping DEVCON this year, and instead is doing Devconnect in Istanbul
https://twitter.com/efdevconnect/status/1649069463797481472?s=46&t=lKsFiPgcngU2KHAM0N6AJQ

They are using this website for reference for donations to earthquake relief:

https://turkiyereliefdao.org/?filtered=true

<img width="1001" alt="Screen Shot 2023-04-20 at 11 13 03 AM" src="https://user-images.githubusercontent.com/14945613/233440144-73550e65-ec56-4e2b-a742-d0137730b28d.png">

We are on it... but only once. I would love for us to add every other project that is up there on to Giveth and ask Turkey Relief DAO to add us as the first drop down option.

Yes, we are asking them to add us first... why? Because we reward the donors, 100% goes to the non-profit and we give the donors  GIVbacks... its a solid argument. 

We also should just add the projects to the dapp. We can reach out too.. but honestly, i'm sad we didn't have all these projects up there already, they all have donation addresses.. 

We can also reach out to this turkey relief DAO and see if there is any other integration points... but given that the entire ethereum Community is going to turkey in november, we should double down on our efforts to be a 1 stop shop for all turkey relief crypto donations.

**Comment by WhyldWanderer** [comment]
Now that Im back from vacation, I can take a closer look at this.

I see two needs here

- [ ] Get these projects on Giveth

- Do we create the projects ourselves? Or do we need to reach out to each org and onboard them?

If we create them ourselves, we could have one wallet that is the owner - similar to Giving Block. Who would be the owner of this wallet? It shouldn't just be one person and having it be a multi-sig would be annoying. Is this ethical? They wouldnt be able to share on their social media to become verified... they wouldnt know that they have this funding portal. I can easily create the projects but it would be nice if we could get these projects to be active users of Giveth. Maybe we create the projects for them and do outreach to try to transfer the projects to them somehow?

- [ ] Ask Turkey Relief DAO to add us as the first drop down option

- Do we have a contact there? They are a DAO.. do we need a contact? Do we need permission to create issues for this type of stuff in their github repo? https://github.com/neu-fi/turkiye-relief-dao/issues Can't we just make an issue that says 'add this' and heres why? I can look into the process that their DAO uses to make decisions... maybe we can create a proposal to get it added there... 


We will need to add all of the projects before doing outreach. Also, if we choose to add the projects on their behalf, how do we create a process for doing this with other projects and efforts similar to this? What does it take to verify that a wallet address actually belongs to a project and how do we maintain the projects that we 'own' if they change wallet addresses etc.. how do we keep up?

**Comment by WhyldWanderer** [comment]
I am working on creating the profile that will own these projects on Giveth. It would be nice to have a cool profile picture to use.

@rodricastillo9013 Do you think you could put something simple together?
Maybe something like this but better:

![image](https://github.com/Giveth/giveth-planning/assets/74267818/30af21de-dc55-4a4a-b226-643f0edfb775)

**Comment by WhyldWanderer** [comment]
The project owner wallet has been created and I am beginning to add the projects to Giveth.
User: https://giveth.io/user/0xd39a9524a7eb914d455a38a80aaf86e03c65bb09

These projects only accept a limited list of tokens on Ethereum so I am working with Carlos to make sure that only these tokens are accepted on the donation page. This data can be found in this spreadsheet:
https://docs.google.com/spreadsheets/d/1GtBNwesVl_Cxr4IYeX1W7t5BHJBmQSu1WBFda35g0ag/edit#gid=0

**Comment by WhyldWanderer** [comment]
Adding another task to the list...

- [ ] Once projects are listed and active, we should add these projects to the blog post here:
https://blog.giveth.io/ways-to-help-earthquake-victims-in-turkey-and-syria-on-giveth-155d7855164

**Comment by WhyldWanderer** [comment]
![Giveth Turkey](https://github.com/Giveth/giveth-planning/assets/74267818/ed31fe1c-e3a7-4af5-8154-20b92643794e)

**Comment by WhyldWanderer** [comment]
All projects have been added. You can find all of the project links in the above spreadsheet.
Now we need to restrict the list of tokens that each one of these projects accepts before listing and verifying them.

I have made an issue to ask the devs to do this since I am unable to in the AdminBro.
Issue: https://github.com/Giveth/giveth-dapps-v2/issues/2733

**Comment by WhyldWanderer** [comment]
Hooray! The token restriction has been figured out and these projects are now live, listed, and verified on the DApp!! 

Now we can move forward with the outreach to Turkey Relief DAO. I am in contact with Mert and he should be able to help us in the outreach efforts. I would also like to sync with @SnakeGH on this initiative as he has experience with strategical partner outreach. 

I will add this to the agenda for the Devconnect call today and add any updates here in the issue.

**Comment by oyealmond** [comment]
Hey @WhyldWanderer! I would love to start building on this effort as our main message on Devconnect, are we going to gamify ppl donating to this projects as with Impact Quests? Not as with a complex dynamic as with impact quest, but I can imagine some sort of price after ppl donating to these projects. 

Are you envisioning something similar?

**Comment by oyealmond** [comment]
Agree @WhyldWanderer! Thanks for your guidance! And for keeping me posted on the outcomes of this conversation.
